schema.md

```diff
@@ -0,0 +1,555 @@
+# `publiccode.yml` standard
+
+This document contains an informal description of the schema of
+`publiccode.yml`.
+
+## Structure
+
+### Country-specific extensions
+
+While the standard is structured to be meaningful on an international
+level, there are additional information that can be added that makes
+sense in specific countries, such as declaring compliance with local
+laws or regulations. The provided extension mechanism is the usage
+of country-specific sections.
+
+All country-specific sections are prefixed with the two-letter ISO 639-1
+country code, followed by a dash (`-`), followed by the section name.
+For instance `it-pianotriennale` is a section declaring compliance to
+the Italian digital transformation plan.
+
+Notice that country-specific keys within international sections are
+not allowed. Countries that want to extend the format should add one
+or many country-specific sections instead.
+
+Documentation for country specific sections is maintained in separate
+files:
+
+* Italy: [Italian extensions](it-schema.md)
+
+
+## Top-level section
+
+### Key `version`
+
+* Type: string
+* Presence: mandatory
+* Example: `"0.1"`
```

Comment:
If we want to SEMVER this then this should be

```yaml
version: "0.1.0"
```

(Also the comment in the example still reads `0.0.1`)

Reply:
I personally don't think full semver spec is relevant. I think what we want is to have a 1.0, evolve it and then maybe break it at some point in the distant future. So I think two numbers are enough.

Reply:
Cool, should we not be at 0.1 though until we actually release this?